ui/apps/platform/src/hooks/usePermissions.ts

```diff
@@ -4,6 +4,7 @@ import { createStructuredSelector } from 'reselect';
 import { selectors } from 'reducers';
 import { Access } from 'types/role.proto';
 import { ResourceName } from 'types/roleResources';
+import { replacedResourceMapping } from 'constants/accessControl';
```

Comment:
Do you need to make the corresponding change in this file:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/reducers/roles.js#L107-L119